test/vol.c

```diff
@@ -558,6 +561,75 @@ reg_opt_datatype_get(void H5_ATTR_UNUSED *obj, H5VL_datatype_get_args_t *args, h
     return ret_value;
 } /* end reg_opt_datatype_get() */
 
+/*-------------------------------------------------------------------------
+ * Function:    fake_vol_info_to_str
+ *
+ * Purpose:     Convert the fake VOL info to a string
+ *
+ * Return:      SUCCEED/FAIL
+ *
+ *-------------------------------------------------------------------------
+ */
+static herr_t
+fake_vol_info_to_str(const void *info, char **str)
+{
+    herr_t    ret_value = SUCCEED; /* Return value */
+    const int val       = *(const int *)info;
+
+    /* Verify the info is correct before continuing */
+    if (val != INT_MAX) {
+        HDprintf("The value of info (%d) is incorrect\n", val);
+        return FAIL;
+    }
+
+    *str = (char *)malloc(16);
+
+    HDsnprintf(*str, 16, "%d", *((const int *)info));
```

Comment:
What is 16?  If you must use numerical value, some comment would be helpful, I think...

Reply:
In this case it's an arbitrary size picked for the VOL connector message, but it's probably not a terrible idea to make sure the malloc and snprintf calls use the same value, even if it's just test code. @raylu-hdf can you update this to use some macro for the magic number here?